### PR TITLE
Init @universal-login/react

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,6 +39,11 @@ jobs:
                 - universal-login-{{ .Environment.CIRCLE_SHA1 }}
       - run: cd universal-login-sdk && yarn test:nobuild
       - run: cd universal-login-relayer && yarn test:nobuild
+  test_react:
```
Comment (JustynaBroniszewska) [inline .circleci/config.yml]: add to jobs

Comment (JustynaBroniszewska) [reply .circleci/config.yml]: And also you need to `restore cache` or run `yarn build`

File: universal-login-react/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "clean": "rm -fr dist",
+    "build": "yarn clean && tsc && tsc -p tsconfig.json",
+    "test": "yarn build && yarn test:nobuild",
+    "test:nobuild": "NODE_ENV=test mocha -t 10000 --exit \"dist/test/**/*.js\""
+  },
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {
```
Comment (JustynaBroniszewska) [inline universal-login-react/package.json]: add types for example: `@types/chai`

File: .circleci/config.yml [hunk 3]
```diff
@@ -39,6 +39,14 @@ jobs:
...
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - restore_cache:
+            keys:
+                - universal-login-{{ .Environment.CIRCLE_SHA1 }}
+      - run: cd universal-login-react && yarn test
   test_wallet_example:
```
Comment (JustynaBroniszewska) [inline .circleci/config.yml]: merge test_react and test_wallet_example

Comment (xitronix) [reply .circleci/config.yml]: 👍

File: universal-login-react/test/test.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import {expect} from 'chai';
+
+describe('Test congiguration', () => {
+  it('should be executed', () => {
+    expect(1).to.be.eq(1);
```
Comment (marekkirejczyk) [inline universal-login-react/test/test.ts]: test div returns test div?

File: universal-login-react/test/test.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import 'jsdom-global/register';
+import React from 'react';
+import {expect} from 'chai';
+import {TestDiv} from '../src/test';
+import {configure, ReactWrapper, mount} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({adapter: new Adapter()});
```
Comment (JustynaBroniszewska) [inline universal-login-react/test/test.tsx]: Why not do it like in [wallet](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/configuration.ts) (as a configuration file, that we require in test script)?

File: universal-login-react/package.json (hunk 2 above)
Comment (marekkirejczyk) [reply universal-login-react/package.json]: 👍 To do in next PR

File: universal-login-react/test/test.tsx (hunk 5 above)
Comment (marekkirejczyk) [reply universal-login-react/test/test.tsx]: 👍 To do in next PR

**Pull request merged**